test/e2e/traceflow_test.go

```diff
@@ -1931,17 +1985,23 @@ func testTraceflowInterNode(t *testing.T, data *TestData) {
 		for _, tc := range testcases {
 			tc := tc
 			t.Run(tc.name, func(t *testing.T) {
-				t.Parallel()
+				if len(clusterInfo.windowsNodes) == 0 {
```

Comment:
Could you add a comment explaining why tests on windows Nodes should not run in parallel?

Reply:
Added comments and TODO here.